index.html

```diff
@@ -44,13 +56,19 @@ <h3>I am a software developer</h3>
                     <li><a href="#"><img src="Normal_Button/Tertiary/icons/Icon-Medium.png" alt="medium"> </a></li>
                    
                 </ul>
+            </div>
+               
                 
         </section>
                 <!-- work section -->
-        <section> 
+        <section class="work-section" id="work"> 
             <div class="work-section-container">
+                <div class="heading">
                 <h2> My Recent Works </h2>
                 <hr>
+
+                </div>
+                
                 
                 <div class="work-container">
                     <div class="item work-item-1">
```

Comment:
- [ ] Please, kindly add the hover, disabled, active CSS pseudo classes to your `see Project button`
- [ ] Add the link to your portfolio live demo view or the link to your portfolio GitHub repository.

![btnlink](https://user-images.githubusercontent.com/59397159/165845622-7de16d55-6778-4d30-9a7d-c2477e0c88c8.png)